lib/src/utils/nom_constants.dart

```diff
@@ -1,5 +1,4 @@
-const znnDecimals = 8;
-const qsrDecimals = 8;
+const coinDecimals = 8;
```

Comment:
Given both values are `8` and probably won't ever change, I don't mind either way.

Note: 
![image](https://github.com/zenon-network/znn_sdk_dart/assets/104875699/ec7348e3-cea9-4cc0-8ceb-72eeeac57089)

Do we feel strongly about keeping these two constants separate?

Reply:
From a principles point of view, there needs to be a good reason to change the public interface and not just because of personal preferences. Breaking changes to the SDKs public interface causes work for others using the SDK, so there should be a clear benefit. Syrius isn't the only client using this SDK.